code/game/gamemodes/wizard/godhand.dm

```diff
@@ -32,6 +32,9 @@
 	attached_spell.attached_hand = null
 	qdel(src)
 
+/obj/item/weapon/melee/touch_attack/dropped()
+	attached_spell.cancel()
```

Comment:
Does this need a `qdeleted(src)` check? I think items get dropped as they're getting deleted, which would make casting a touch spell instantly reset the cooldown...